activesupport/lib/active_support/core_ext/digest/uuid.rb

```diff
@@ -49,5 +51,11 @@ def self.uuid_v5(uuid_namespace, name)
     def self.uuid_v4
       SecureRandom.uuid
     end
+
+    def self.pack_uuid(uuid)
+      uuid.scan(/(\h{8})-(\h{4})-(\h{4})-(\h{4})-(\h{4})(\h{8})/).flatten.map { |s| s.to_i(16) }.pack("NnnnnN")
```

Comment:
This regex should be anchored so that it only accepts a UUID, rather than any string containing a UUID:

```suggestion
      uuid.match(/\A(\h{8})-(\h{4})-(\h{4})-(\h{4})-(\h{4})(\h{8})\z/).captures.map { |s| s.to_i(16) }.pack("NnnnnN")
```

Reply:
Hi @eugeneius. Thank you for your review. I've accepted the suggested change.